### ISSUE TITLE
VirtualBox guest additions not working

### ISSUE THREAD

**Issue by pkgdemon** [furybsd/furybsd-livecd#133]
![virtualbox guest additions](https://user-images.githubusercontent.com/4109732/80829388-0821dd80-8bb5-11ea-830a-1aade3115414.PNG)

**Comment by Tatsh** [comment]
I am also getting this issue. I was able to install with _VBoxVGA_ but when I rebooted I got a black screen with SDDM. I also tried VBoxSVGA enabled.

Then I installed _xf86-video-vmware_ and enabled VMSVGA, along with deleting `/etc/X11/xorg.conf`. Still a black screen on SDDM.

**Comment by jhfoo** [comment]
+1 on this issue. Replicated on VirtualBox 5.x and 6.1.

**Comment by dickbosman** [comment]
Virtualbox guest editions work perfectly with the VBoxSVGA driver. Take care NOT to activate 3D or else it will not work.

**Comment by Tatsh** [comment]
> Virtualbox guest editions work perfectly with the VBoxSVGA driver. Take care NOT to activate 3D or else it will not work.

So what you mean to say is the XFCE version works. The KDE version is the one that doesn't work in VirtualBox.

**Comment by jhfoo** [comment]
With @dickbosman's confirmation was inspired to try installing XFCE again. Basic installation works! I suspect having the user join video and webcamd (why webcamd?) groups also enabled automatic screen resizing as well.   

Continuing to install other stuff to see if I can finally use Fury as my dev platform!

**Comment by Tatsh** [comment]
I think it's worth sending a bug report to VirtualBox about the application showing 'Invalid settings detected' when using VBoxSVGA. Maybe this affects FreeBSD too.

![image](https://user-images.githubusercontent.com/724848/83366310-32b1b280-a37c-11ea-9438-0ca8c96746ab.png)

**Comment by pkgdemon** [comment]
Before the Q2 2020 image was released I unfortunately only tested this with VirtualBox 5 and it worked.  VirtualBox 6 appears to introduce the switch to VMware SVGA adapter.  Simply changing the setting for graphics controller to VBoxSVGA as shown above corrects it for me and yes I can confirm I see invalid settings when I chose that but it works for me.

I suspect Oracle may intend to stop shipping guest additions for FreeBSD perhaps?  I can't think of any other reason VMware's driver would be the default, and selecting VirtualBox driver would warn of invalid settings.  Either way I should be able to automate this for the next image in July to use xf86-video-vmware by also making a condition to check for that with pciconf so that if VMSVGA is selected instead it will work.
